docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -137,20 +144,26 @@ Run the script that will create the necessary accounts in noobaa and update the
 node ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_setup.js
 ```
 
-Note: If you want to ignore PythonDeprecationWarnings use (which will then ignore all Python warnings, so keep that in mind):
-```bash
-export PYTHONWARNINGS="ignore"
-```
+Note: If you want to ignore PythonDeprecationWarnings add the following flag to the test command (which will then ignore all Python warnings, so keep that in mind):
+`-- --disable-pytest-warnings`
+
+note that every flag that comes after `--` is past to pytest from tox. so if there is already `--` in the command just put `--disable-pytest-warnings` as part of the flags after it, no need to add another `--` notation.
+
+for example to add --disable-pytest-warnings to the command:
+`S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- -m 'not fails_on_aws' ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage`
+
+it should be:
+`S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- -m 'not fails_on_aws --disable-pytest-warnings ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage`
```

Comment:
I'll check, but I don't think we have any example for that in the document, I still think this explanation should be here since people might look up different commands in the s3-tests repo, which does not explain this. maybe I'll use a simpler example, and explain it in our document as well

Reply:
looks like this is the only case we use this, and even this case is only used in the tests scripts. I have added explanation for it in the run all tests section. this explanation is based on the one given in the s3-tests repos readme. however, since both of those sections will probably not be used by the average tester. I am not sure if it just makes it more confusing and I should leave both of them out. what do you think?

Reply:
It is a great explanation.
When we finish with everything, we can rethink maybe placing it somewhere else in the doc.